src/protocols/secure_channel/CASESession.cpp

```diff
@@ -339,9 +339,6 @@ CHIP_ERROR CASESession::SendSigma1()
     uint8_t destinationIdentifier[kSHA256_Hash_Length] = { 0 };
 
     // Generate an ephemeral keypair
-#ifdef ENABLE_HSM_CASE_EPHEMERAL_KEY
-    mEphemeralKey.SetKeyId(CASE_EPHEMERAL_KEY);
-#endif
```

Comment:
Why was CASE touched?

Reply:
This code belongs to function - SendSigma1 (commissioner side).. Since HSM integration is targeted only for end nodes for now, this is not required. It was a mistake from earlier PRs.

Reply:
What about OTA?  What about binding?  There are reasons for constrained nodes to initiate sessions.

Reply:
Yes. I agree. Even that part of the integration will also be considered in the subsequent PRs. Removing the change as it is a incomplete integration.